k8s/object.bzl

```diff
@@ -170,12 +170,14 @@ def _common_impl(ctx):
   if namespace_arg:
     namespace_arg = "--namespace=\"" +  namespace_arg + "\""
 
+  
   substitutions = {
       "%{cluster}": cluster_arg,
       "%{context}": context_arg,
       "%{user}": user_arg,
       "%{namespace_arg}": namespace_arg,
       "%{kind}": ctx.attr.kind,
+      "%{kubectl_args}": " ".join(ctx.attr.kubectl_args or []),
```

Comment:
This seems kinda hacky, but I cannot use `ctx.expand_make_variables` since that expects a string.  Also ` ctx.expand_make_variables` I think is deprecated now.

Reply:
you should be able to execute ctx.expand_make_variables on each of the elements in kubectl_args. i.e. something like:
```
kubectl_args = ctx.expand_make_variables(kubectl_arg)  for kubectl_arg in ctx.attr.kubectl_args 
...
"%{kubectl_args}": " ".join(kubectl_args)
```